src/pug/blog/year_one_recap.pug

```diff
@@ -0,0 +1,67 @@
+extends _blog.pug
+
+block append subtitle
+  | Year one recap
+
+block content
+  h2 Year one recap
+  +news_header('22 November 2023', 'report_18', '')
+
+  p One year ago Input Labs went live, it is time to review our achievements and our failures.
+
+  h3 Things that went nicely
+  h4 Community
+  ul
+    li Our nice #[em community kept growing], now with 735 discord members. This is by far the thing that makes us more proud, and it is awesome to see people helping each other there. To make it even better, some of the members are experts in their fields (electrical engineers, industrial designers) and contribute with expert advice and knowledge.
+    li We got a #[em small office] in the #[+ax('https://maria.io/about/') Maria 01 startup campus] in Helsinki, and we moved our 3D-printers and other tools there, making it the "Lab" of Input Labs.
+    li The project received plenty of #[em positive feedback] in the different social channels in which we have presence, and lot of people encouraged us to continue. We got very limited (almost negligible) negative interactions.
+    li About #[em 30 community mods] were created for the Alpakka controller, most of them modifications of the 3D-printed models for different purposes.
+    li Our project got supported by #[em 10 wonderful folks in Patreon].
+    li We opened the #[em distribution channel (shop)] to better serve testers, reviewers, and everyone without the tools to build a controller from scratch. Setting up the logistics to build and deliver all these components has been a challenge, but also a success in verifying that the "DIY-or-buy" model for supporting open source projects can work for us.
+
+  h4 Exposure
+  ul
+    li We published #[+ax(org.youtube) 15 videos on Youtube], reaching a total of 58K views.
+    li The Alpakka controller was independently #[em reviewed by 5 Youtube channels].
+    li We were featured in the #[+ax('https://magpi.raspberrypi.com/issues/128') Raspberry Pi magazine], Heise magazine, and at least 10 other tech-related websites.
+    li The Alpakka controller was demoed among a few #[em videogame companies] with office in Helsinki, and had a small demo booth at the #[+ax('https://igda.org') IGDA] (International Game Developers Association) event.
+
+  h4 Development
+  ul
+    li There were #[em 21 firmware releases] (including betas), addressing lot of community requested features such as:
+    ul
+      li #[em Console Legacy] profile which makes easier to play emulators and retro games.
+      li #[em Racing] profile which allows to use the controllers as a steering wheel.
+      li #[em Generic] gamepad mode (aka DirectInput) implementation, which is useful both for retro gaming and for simulators expecting generic joysticks or steering wheels.
+      li Full #[em Steam Deck] compatibility.
+      li We created the #[em Ctrl app] as an alternative (and optional) method for configuring the Alpakka controller.
```

Comment:
This should be separate. It doesn't fit with the rest of the list and is not just a firmware feature.
Add it as a separate line "We also implemented a more advanced method of configuring the Alpakka, based Web-USB and the Ctrl web-app.